presto-main/src/main/java/com/facebook/presto/sql/planner/LocalExecutionPlanner.java

```diff
@@ -1772,6 +1784,22 @@ else if (comparisonOperator.get() == OperatorType.LESS_THAN_OR_EQUAL) {
             throw new UnsupportedOperationException("Unsupported spatial function: " + functionName);
         }
 
+        private SpatialPredicate sphericalSpatialTest(QualifiedFunctionName functionName, Optional<OperatorType> comparisonOperator)
+        {
+            if (functionName.equals(ST_DISTANCE)) {
+                if (comparisonOperator.get() == OperatorType.LESS_THAN) {
```

Comment:
Instead of repeating .get() pull it onto the stack and rename the parameter so it doesn't conflict? Don't feel strongly about it, but it feels a hair nicer.

Reply:
It's a little tricky, because `comparisonOperator` is empty for a bunch of cases, so we can only extract it out a little.  I'm tempted to leave it as is.